inst/include/frogger.hpp

```diff
@@ -30,7 +30,51 @@ struct Leapfrog {
     State initial_state = {};
     initial_state.reset();
     if constexpr (ModelVariant::run_demographic_projection) {
-      initial_state.dp.p_total_pop = pars.dp.base_pop;
+
+      // Initialise base year ouputs
+      const auto& p_dp = pars.dp;
+      auto& is_dp = initial_state.dp;
+      int t0 = 0;
```

Comment:
should this be set as a state space default rather than hardcoded here?

Reply:
Had a similar question when I was writing it. @r-ash